### ISSUE TITLE
[BUG] Scroll on kirby-page generate many change-detection cycles

### ISSUE THREAD

**Issue by mictro** [kirbydesign/designsystem#2939]
## Describe the bug
On the 11th of March @NikolajSkousen wrote in kirby-guild:
_Hej Kirby,
Når jeg profilerer appen (angular profiler) er den begyndt at melde et ret voldsomt antal change detection cycles når man scroller på sider. Jeg kan se det samme gør sig gældende på kirby-page på jeres cookbook.
I december er der blevet tilføjet noget kode i page.component, som ser ud til at være udslagsgivende. Har vedlagt en optagelse, hvor i kan se profileringen med/uden den stump kode, hvor jeg scroller på jeres cookbook:_

Part1:
https://user-images.githubusercontent.com/18654594/224692802-d8054978-8d47-4986-b5c2-33ef16d49391.mov

Part2:
https://user-images.githubusercontent.com/18654594/224692829-25299432-0e1d-4316-89b2-fa344748d0f3.mov

[Kirby Guild Entry](https://bankdata.slack.com/files/U023AGHUQLF/F04TVFQ0W5P/screen_recording_2023-03-10_at_23.53.37.mov?origin_team=T0ADQ12PK&origin_channel=CGKJ1RBM4)

## Describe how to reproduce the bug
See part1 + part2 video above.
-the angular profiler is available as part of the Angular DevTools Chrome-extension [Download Link](https://chrome.google.com/webstore/detail/angular-devtools/ienfalfjdbdpebioblfackkekamfmbnh)

1. Go to the cookbook page 'simple page' 
2. Open the 'Angular' profiler and start recording
3. Scroll page content up and down 
4. See that the number of change-detection cycles grows rapidly.

## Which Kirby version was used?
8.1.3

## What was the expected behavior?
Fewer change-detection cycles.

## Add any screenshots
See video above...

## Please complete the following information:
- OS: MacOS
- Browser Chrome
- Browser version latest

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [x] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by mictro** [comment]
This bug report is a duplicate of #2940
